python/README.md

```diff
@@ -50,7 +50,47 @@ object_name = 'file1'
 data = sc.textFile(cos.url(object_name, bucket_name))
 ```
 
-### Bluemix / Data Science Experience
+### Bluemix IBM Cloud Object Storage (COS)
+The class CloudObjectStorage allows you to connect to an IBM bluemix COS. You can connect to a bluemix COS using api keys
+as follows:
+
+```python
+import ibmos2spark
+
+credentials = {
+    'endpoint': 'XXX',
+    'api_key': 'XXX',
+    'service_id': 'XXX'
+}
+
+configuration_name = 'os_bluemix_cos_config'
+cos = ibmos2spark.CloudObjectStorage(sc, credentials, configuration_name, 'bluemix_cos')
```

Comment:
Of course having the args name would work but i think this form is quite nicer and minimised. The user who is using our lib knows Python and knows how params work in it. I think this form is fine especially because we explained in details all params and what they do.